CefSharp.Core/Internals/StringUtils.h

```diff
@@ -117,6 +117,25 @@ namespace CefSharp
                     cef_string_copy(pStr, str->Length, &cefStr);
                 }
             }
+
+            /// <summary>
+            /// Creates a detailed expection string from a provided Cef V8 exception.
+            /// </summary>
+            /// <param name="exception">The exception which will be used as base for the message</param>
+            [DebuggerStepThrough]
+            static CefString CreateExceptionString(CefRefPtr<CefV8Exception> exception)
+            {
+                if (exception.get())
+                {
+                    String^ message = ToClr(exception->GetMessage()) + "\n@ " +
```

Comment:
Would be nicer to create the native `CefString` rather than convert to `String` and then back again.